src/gui/dialogs/DialogStateful.hpp

```diff
@@ -80,12 +75,12 @@ class IDialogStateful : public IDialog {
     bool Change(uint8_t phs, uint8_t progress_tot, uint8_t progress); // = 0; todo should be pure virtual
     virtual ~IDialogStateful();
 
-    static constexpr rect_ui16_t get_radio_button_size() {
-        rect_ui16_t rc_btn = { 0, 32, 240, 320 - 96 }; //msg box size
-        rc_btn.y += (rc_btn.h - 40);                   // 30pixels for button (+ 10 space for grey frame)
-        rc_btn.h = 30;
-        rc_btn.x += 6;
-        rc_btn.w -= 12;
+    static rect_ui16_t get_radio_button_size() {                                // cannot be const(expr)
```

Comment:
I'm still not happy with removing constexpr (bigger code, less efficient)
If you insist, at least put definition of the method into cpp file.
And comment it with todo or fixme to be changed to constexpr after gui_defaults refactoring